lib/travis/build/script/php.rb

```diff
@@ -100,7 +100,11 @@ def cache_slug
         private
 
         def version
-          config[:php].to_s
+          if config[:hhvm]
+            "hhvm-#{config[:hhvm].to_s}"
```

Comment:
What would be the equivalent of:

```
php:
 - hhvm
 - hhvm-nightly
```

i.e. how would I specify that one of the things that I want to run is the latest version of HHVM?

Reply:
@fredemmott Thanks for chiming in!

How about if we supported `latest` in that case e.g. the equivalent of your snippet with `language: hack` would be

```yaml
hack:
  - latest
  - nightly
```

What do you think?

Note: the current state of the PR doesn't support this yet.